### DISCUSSION TITLE
Show spinner rpm & unstable rate

### DISCUSSION THREAD

**Discussion by EtaoinWu** [ppy/osu#21486] [Ideas]
Currently in stable, you can see the average spinning rpm and unstable rate for spinners when hovering on the hp graph after a play/replay. However, the "error bar" UI in lazer does not display spinner information.

Would it be possible to display spinner stats in lazer?

**Comment by peppy** [comment]
Should definitely be displayed, somewhere in the extended statistics area as its own section. This is already quite easy to make happen.
